x-pack/plugins/security_solution/server/lib/detection_engine/rule_execution_log/rule_execution_log_client.ts

```diff
@@ -56,41 +57,56 @@ export class RuleExecutionLogClient implements IRuleExecutionLogClient {
 
   /** @deprecated */
   public find(args: FindExecutionLogArgs) {
-    return this.client.find(args);
+    return withSecuritySpan('RuleExecutionLogClient.find', () => this.client.find(args));
   }
 
   /** @deprecated */
   public findBulk(args: FindBulkExecutionLogArgs) {
-    return this.client.findBulk(args);
+    return withSecuritySpan('RuleExecutionLogClient.findBulk', () => this.client.findBulk(args));
   }
 
   public getLastFailures(args: GetLastFailuresArgs): Promise<IRuleStatusSOAttributes[]> {
-    return this.client.getLastFailures(args);
+    return withSecuritySpan('RuleExecutionLogClient.getLastFailures', () =>
+      this.client.getLastFailures(args)
+    );
   }
 
   public getCurrentStatus(
     args: GetCurrentStatusArgs
   ): Promise<IRuleStatusSOAttributes | undefined> {
-    return this.client.getCurrentStatus(args);
+    return withSecuritySpan('RuleExecutionLogClient.getCurrentStatus', () =>
+      this.client.getCurrentStatus(args)
+    );
   }
 
   public getCurrentStatusBulk(args: GetCurrentStatusBulkArgs): Promise<GetCurrentStatusBulkResult> {
-    return this.client.getCurrentStatusBulk(args);
+    return withSecuritySpan('RuleExecutionLogClient.getCurrentStatusBulk', () =>
+      this.client.getCurrentStatusBulk(args)
+    );
   }
 
-  public deleteCurrentStatus(ruleId: string): Promise<void> {
-    return this.client.deleteCurrentStatus(ruleId);
+  public async deleteCurrentStatus(ruleId: string): Promise<void> {
+    await withSecuritySpan('RuleExecutionLogClient.deleteCurrentStatus', () =>
```

Comment:
Why are we `await`'ing here now and weren't before?

Reply:
I removed `return`, and added `async` + `await`. Don't remember why I did that, tbh. But these functions are identical:

```ts
const foo = async () => {
  await asyncFnReturningVoid();
};

const bar = () => {
  return asyncFnReturningVoid();
};
```